index.js

```diff
@@ -1,40 +1,31 @@
-module.exports = function(filteredImports) {
-  return function(babel) {
-    // A stack of booleans that determine whether an expression statement
-    // should be removed as it is exited. Expression statements are removed
-    // when they contain a reference to a filtered imported.
-    var shouldRemove;
-
-    return new babel.Transformer('babel-plugin-filter-imports', {
-      Program: {
-        enter: function() {
-          shouldRemove = [];
-        },
-        exit: function() {
-          shouldRemove = undefined;
-        }
-      },
```

Comment:
Can you add a test for multiple plugin instances? Babel interleaves plugins so I'm pretty sure it will be wildly broken unless we store `shouldRemove` in `state`. i.e.

```
      Program: function(path, state) {   
        state.shouldRemove = []; 
      },
```

and access it via `state` in all the visitor handlers.